.github/workflows/msbuild.yml

```diff
@@ -58,5 +58,7 @@ jobs:
         run: dotnet build dafny.msbuild/Test -t:VerifyDafny
       - name: Test VerifyDafny task (sequential)
         run: dotnet build dafny.msbuild/Test -t:VerifyDafny -p:VerifyDafnyJobs=1
+      - name: Test VerifyDafny task (sequential with definite assignment)
+        run: dotnet build dafny.msbuild/Test -t:VerifyDafny -p:VerifyDafnyJobs=1 -p:VerifyDafnyPassthrough='"definiteAssignment:3"'
```

Comment:
Are both layers of extra quotes necessary?

Reply:
This is a result of https://stackoverflow.com/questions/11987384/how-do-i-pass-a-property-value-containing-a-semicolon-on-the-msbuild-command-lin?rq=1